src/ui-kit/experimental/hierarchical/hierarchical-tree-grid/hierarchical-tree-grid.component.ts

```diff
@@ -42,60 +40,60 @@ export class SamHierarchicalTreeGridComponent implements OnInit {
   @Output() selectResults = new EventEmitter<object[]>();
 
   /**
-   * 
+   * Columns to be displayed
    */
   public displayedColumns = ['select'];
 
   /**
-   * 
+   * column fields that will display for each column 
    */
   public columnFieldName = [];
 
   /**
-   * 
+   * Column header text list
    */
   public columnHeaderText = [];
 
   /**
-   * 
+   * List of the items selected by the checkboxes or the radio buttons
    */
   public selectedList: object[] = [];
 
   /**
-   * 
+   * Hierarchical data source
    */
   public hierarchicalDataSource: HierarchicalDataSource | null;
 
   /**
-   * 
+   * event called when data changed
    */
   public dataChange: BehaviorSubject<object[]> = new BehaviorSubject<object[]>([]);
 
   /**
-   * 
+   * Selection mode default is checkbox
    */
   selectionMode: string = "checkbox";
 
   /**
-   * 
+   * Sort Directive
    */
   @ViewChild(SamSortDirective) sort: SamSortDirective;
 
   /**
-   * 
+   * constructor
```

Comment:
Can you drop the comments for constructor, ngOnChanges, etc? The comment is duplicating the name of the method.